pkg/agent/sidecar/service/observer/observer.go

```diff
@@ -421,8 +421,8 @@ func (o *observer) backup(ctx context.Context) (err error) {
 		}
 	}()
 
-	log.Infof("backup directory %s started", o.dir)
-	defer log.Infof("backup directory %s finished", o.dir)
+	log.Infof("started to backup directory %s", o.dir)
+	defer log.Infof("finished to backup directory %s", o.dir)
```

Comment:
This log will be printed even an error occurred. Is it the intended usage?
If no we should move this log message to the end of this function instead of using defer here?

Reply:
It is intended usage. However as you pointed out it makes misunderstandings. We're going to move this line to the end.